workspaces/api-gateway/src/api/products/services/UpdateProductService.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import type { Repository } from 'typeorm';
+
+import { Product } from 'database/models/product.entity';
+import { ProductsCategories } from 'database/models/productsCategories.entity';
+import { CheckProductExistedService } from 'api/products/services/CheckProductExistedService';
+import { CreateProductCategoryService } from 'api/products/services/CreateProductCategoryService';
+import { createSlug } from '../../../utils';
```

Comment:
why don't we use absolute path here?

Reply:
chỗ này ban đầu em dùng absolute path mà lúc chạy test bị lỗi nên em phải đổi qua relative path mới chạy được ạ